### ISSUE TITLE
require_once C_BEGIN_TEMPLATE;

### ISSUE THREAD

**Issue by ncopz** [enikeishik/ufocms2#1]
Warning: require_once(C:/xampp/htdocs/install/begin.php): failed to open stream: No such file or directory in C:\xampp\htdocs\cms\install\index.php on line 194

Fatal error: require_once(): Failed opening required 'C:/xampp/htdocs/install/begin.php' (include_path='\xampp\php\PEAR') in C:\xampp\htdocs\cms\install\index.php on line 194

**Comment by enikeishik** [comment]
Folder install contains begin.php file in repository (https://github.com/enikeishik/ufocms2/tree/master/install). May be your package is broken?

**Comment by ncopz** [comment]
I don't think so, because I got it from https://github.com/enikeishik/ufocms2

**Comment by enikeishik** [comment]
So look at you folder C:/xampp/htdocs/install, is begin.php file in there?

**Comment by ncopz** [comment]
![begin](https://user-images.githubusercontent.com/26650405/106869392-75453f00-6702-11eb-932a-b941edc9a22b.png)
there is..

**Comment by enikeishik** [comment]
So it not cms trouble, your PHP can't include existing file.

**Comment by ncopz** [comment]
Then what is the solution while I am using the latest version of xampp.
